draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -755,31 +755,31 @@ An elliptic curve E is specified by a cubic equation in two variables and a
 finite field F. An elliptic curve equation takes one of several standard forms,
 including (but not limited to) Weierstrass, Montgomery, and Edwards.
 
-The curve E forms an algebraic group whose elements are the points (x, y)
-satisfying the curve equation, where x and y are elements of F. This group
-has order n, meaning that there are n distinct points (x, y). In general,
-security of cryptographic primitives requires using a group of prime order.
-However, not all elliptic curves induce groups of prime order: most elliptic
-curves have order n = h * r, where r is a large prime and h is an
-integer called the cofactor. Thus, for cryptographic applications a hash
-function to the curve E should return points in the subgroup of order r. For
-a point not in the prime-order subgroup, the process of mapping to a point
-in the prime-order subgroup is called clearing the cofactor; we discuss this
-process in {{cofactor-clearing}}.
-
-Certain encoding functions restrict the form of the curve equation, the characteristic
-of the field, and/or the parameters of the curve. For each encoding,
-this document lists the relevant restrictions.
+The curve E forms an algebraic group, whose elements are the
+points (x, y) satisfying the curve equation, where x and y are elements of F.
+This group has order n, meaning that there are n distinct points.
+
+For security reasons, groups of prime order MUST be used. Elliptic curves
+induce subgroups of prime order. Let G be a subgroup of the curve of prime
+order r, where n = h * r.
+In this equation, h is an integer called the cofactor.
+The process of mapping an elliptic curve point to a point in G is
+called clearing the cofactor, this operation is described in {{cofactor-clearing}}.
```

Comment:
nit: comma splice. I suggest either changing the comma to semicolon, or changing to a period and making "This operation..." a new sentence.